experimental/uefi/app/Cargo.toml

```diff
@@ -16,6 +16,9 @@ log = { version = "*" }
 ciborium = { version = "*", default-features = false }
 ciborium-io = "*"
 oak_remote_attestation = { path = "../../../remote_attestation/rust" }
+oak_remote_attestation_sessions = { path = "../../../remote_attestation_sessions" }
+lru = "*"
```

Comment:
Is this still required now that the session crate is used?

Reply:
It's not, thanks for catching. Usually xtask flags unused deps, but due to #2654 it doesn't yet do it for the UEFI app. :)